pkg/server/singleprocess/service_project.go

```diff
@@ -79,6 +79,23 @@ func (s *Service) ListProjects(
 	return &pb.ListProjectsResponse{Projects: result}, nil
 }
 
+func (s *Service) DestroyProject(
+	ctx context.Context,
+	req *pb.DestroyProjectRequest,
+) (*empty.Empty, error) {
+	if err := serverptypes.ValidateDestroyProjectRequest(req); err != nil {
+		return nil, err
+	}
+
+	err := s.state(ctx).ProjectDelete(req.Project)
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: Do we need to return something other than empty?
+	return &empty.Empty{}, nil
```

Comment:
In this case, maybe not? Where do we queue the DestroyOperation? In that case we should return the job ID associated with destroying the resources. But since it looks like this simply deletes Project from the DB, it's safe to return empty.